typescript/test/testdata/keymanager/src/v1/key_management_service_client_config.json.baseline

```diff
@@ -16,124 +14,101 @@
           "retry_delay_multiplier": 1.3,
           "max_retry_delay_millis": 60000,
           "initial_rpc_timeout_millis": 20000,
-          "rpc_timeout_multiplier": 1.0,
+          "rpc_timeout_multiplier": 1,
           "max_rpc_timeout_millis": 20000,
           "total_timeout_millis": 600000
         }
       },
       "methods": {
         "ListKeyRings": {
-          "timeout_millis": 60000,
```

Comment:
It seems like all of the methods marked non_idempotent are losing their timeout. I'm unsure about this specific test if that is intended or not.

Reply:
That's because these two baseline tests (showcase and KMS) don't have gRPC service config. Previously, `timeout_millis` was hardcoded here.

The `texttospeech` baseline has gRPC service config and `timeout_millis` is set correctly.

Reply:
Ah ok thank you for the clarification.